modyn/supervisor/internal/pipeline_executor/pipeline_executor.py

```diff
@@ -0,0 +1,390 @@
+import json
+import logging
+import multiprocessing as mp
+import os
+import pathlib
+import traceback
+from time import sleep
+from typing import Any, Optional
+
+# import enlighten
```

Comment:
I don't think we'll need enlighten here, it will only be at the client

Reply:
removed.